zign/api.py

```diff
@@ -43,6 +44,13 @@ def __str__(self):
         return 'Configuration error: {}'.format(self.msg)
 
 
+def open_browser(url=None):
+    user = os.getenv('USER') 
+    if os.getegid() == 0 and user and user != "root":
```

Comment:
This looks like a hack, why would we want to support that?

Reply:
just a proposal, removed this part from PR.